ontology/yaml/resources/HVAC/entity_types/AHU.yaml

```diff
@@ -3566,4 +3566,24 @@ AHU_US_SVL_MAT2_4:
   - supply_fan_run_command
   - supply_fan_speed_frequency_sensor
   - supply_fan_speed_percentage_command
-  - supply_air_flowrate_sensor
\ No newline at end of file
+  - supply_air_flowrate_sensor
+
+AHU_STDSPC_SSPC_EFFC_FDPM3X_RTM_RHM:
+  description: "Non-Standard (Swegon) Multi-Zone AHU with thermal wheel, isolation damper and filter pressure monitoring"
+  implements:
+  - AHU
+  - STDSPC
+  - SSPC
+  - EFFC
+  - FDPM3X
+  - RTM
+  - RHM
+  uses:
+  - compressor_speed_percentage_sensor
+  - exhaust_air_temperature_sensor
+  - exhaust_fan_run_status
+  - exhaust_fan_speed_percentage_sensor
+  - isolation_damper_status
```

Comment:
which isolation damper?

Reply:
Changed to outside_air_isolation_damper and exhaust_air_isolation_damper in new commit